test/unit/calculators/business_coronavirus_support_calculator_test.rb

```diff
@@ -146,13 +155,13 @@ class BusinessCoronavirusSupportFinderCalculatorTest < ActiveSupport::TestCase
           assert_not @calculator.show?(:nursery_support)
         end
 
-        should "return false when no non domestic property" do
+        should "return false when no non-domestic property" do
           @calculator.non_domestic_property = "none"
           assert_not @calculator.show?(:nursery_support)
         end
 
         should "return false when not supported business sector" do
-          @calculator.sectors = %w[retail hospitality leisure]
+          @calculator.sectors = %w[retail_hospitality leisure]
```

Comment:
```suggestion
          @calculator.sectors = %w[retail_hospitality_or_leisure]
```

Reply:
These loose option strings are definitely a pain point of this app!

Reply:
Argh good catch. Yes, I find the fragility of the strings annoying and concerning. I did have a look and see if we could maybe use the enum functionality from Rails so they're more restricted but it turns out it's baked into ActiveRecord rather than ActiveModel so hard to use.